Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +9,25 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (IsNullOrEmpty(array)) return;
+
+            bool isSorted;
+            do
+            {
+                isSorted = true;
+                for (int i = 0; i < array.Length - 1; i++)
+                {
+                    if (array[i] > array[i + 1])
+                    {
+                        int oldIndexValue = array[i + 1];
```

Comment:
How many cups do you need to move wine from cup 1 to cup 2?

Reply:
3 is enough